### ISSUE TITLE
bold syntax

### ISSUE THREAD

**Issue by d3matt** [luochen1990/rainbow#144]
Is there currently an option to make the colors bold in addition to the rainbow colors?

**Comment by luochen1990** [comment]
Yes, look at the README about configuration: https://github.com/luochen1990/rainbow#configure
there is `guis` and `cterms` option.

**Comment by d3matt** [comment]
Ok, I'm probably missing something (or have some odd interaction with the normal syntax highlighter).  I set this in my vimrc:
```
let g:rainbow_conf = {'guis': ['bold']}
let g:rainbow_active = 1
```
It seems to be working in XML files but not c++ files (which is what I'm trying to get...)

**Comment by luochen1990** [comment]
emmm, Can you use the tools in the README about [troubleshooting](https://github.com/luochen1990/rainbow#troubleshooting) to check what happend?  It is possible that other syntax rule takes precedence than rainbow.

**Comment by rwmitchell** [comment]
What did you discover, d3matt ?  Did you solve your issue and get bold parens working?
